changelog.d/11675.feature

```diff
@@ -0,0 +1 @@
+Adds a flag to the `synapse_review_recent_signups` script to ignore and filter appservice users.
```

Comment:
```suggestion
Add a flag to the `synapse_review_recent_signups` script to ignore and filter appservice users.
```
We write changelogs in passive tone (imagine the word 'This change, if applied, will …' in front of your message and see if it makes grammatical sense).